src/components/Navigation.tsx

```diff
@@ -66,64 +71,66 @@ function Navigation() {
-        </a>
-      </AllCategory>
-      <NavList>
-        <NavItem active={pathname === '/'}>
-          <NavLink href="/" className={pathname === '/' ? 'active' : ''}>
-            Home
-          </NavLink>
-        </NavItem>
-        <NavItem active={pathname === '/about'}>
-          <NavLink
-            href="/about"
-            className={pathname === '/about' ? 'active' : ''}
-          >
-            About
-          </NavLink>
-        </NavItem>
-        <NavItem active={pathname === '/shop'}>
-          <NavLink
-            href="/shop"
-            className={pathname === '/shop' ? 'active' : ''}
-          >
-            Shop
-          </NavLink>
-        </NavItem>
-        <NavItem active={pathname?.startsWith('/blog')}>
-          <NavLink
-            href="/blog"
-            className={pathname?.startsWith('/blog') ? 'active' : ''}
-          >
-            Blog
-          </NavLink>
-        </NavItem>
-        <NavItem active={pathname === '/contact'}>
-          <NavLink
-            href="/contact"
-            className={pathname === '/contact' ? 'active' : ''}
-          >
-            Contact
-          </NavLink>
-        </NavItem>
-      </NavList>
-      <NavList>
-        <NavItem active={false}>
-          <Image src={heartIcon} alt="heart" />
-        </NavItem>
-        <NavItem active={false}>
-          <Image src={bagIcon} alt="bag" />
-        </NavItem>
-        <NavItem active={false}>
-          <Image src={profileIcon} alt="profile" />
-        </NavItem>
-      </NavList>
-    </Nav>
+    <Container>
+      <Nav>
+        <AllCategory>
+          <a>
+            <Image src={menuIcon} alt="menu" />
+            All Category
+          </a>
+        </AllCategory>
+        <NavList>
+          <NavItem active={pathname === '/'}>
+            <NavLink href="/" className={pathname === '/' ? 'active' : ''}>
+              Home
+            </NavLink>
+          </NavItem>
+          <NavItem active={pathname === '/about'}>
+            <NavLink
+              href="/about"
+              className={pathname === '/about' ? 'active' : ''}
+            >
+              About
+            </NavLink>
+          </NavItem>
+          <NavItem active={pathname === '/shop'}>
+            <NavLink
+              href="/shop"
+              className={pathname === '/shop' ? 'active' : ''}
+            >
+              Shop
+            </NavLink>
+          </NavItem>
+          <NavItem active={pathname?.startsWith('/blog')}>
+            <NavLink
+              href="/blog"
+              className={pathname?.startsWith('/blog') ? 'active' : ''}
+            >
+              Blog
+            </NavLink>
+          </NavItem>
+          <NavItem active={pathname === '/contact'}>
+            <NavLink
+              href="/contact"
+              className={pathname === '/contact' ? 'active' : ''}
+            >
+              Contact
+            </NavLink>
+          </NavItem>
+        </NavList>
```

Comment:
NavList의 컴포넌트들이 반복되고 있는데 Array로 만들어서 map으로 순회하는 방식은 어떨까요?
```suggestion
       <NavList>
        {list.map((item, index) => (
          <NavItem key={index} active={pathname === item.href}>
              <NavLink href={item.href} className={pathname === item.href ? 'active' : ''}>
                Home
              </NavLink>
            </NavItem>
        ))}
       </NavList>
```